### PR TITLE
Add OCaml support

### PR DESCRIPTION
**Pull request by A-Hilaly**
Adding OCaml support ( #190 )

I use temporarily an image taged `add-ocaml-support`.

Here is the logs after running all ocaml solutions:
screenshot 2018-10-26 at 00 14 30

### CODE REVIEW

**Comment by A-Hilaly** [comment]
Now i have to replace the default image (taged `latest`) with the new one, and fix the `docker-compose.yml`. Any remarks ?

File: Problem006/OCaml/solution_1.ml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+let rec pow n x =
+  if n=0 then 1. else x *. pow (n-1) x;;
+
+let f x = pow 2 (float_of_int ((x * (x + 1)) / 2))
+let ff x = (x * (x + 1) * ((2 * x) + 1)) / 6
+
+let solve n = -(ff n) + (int_of_float (f n))
+
+let __kappa__ =
+solve 100;;
```
[the remark this answers is not in the document]
Comment (A-Hilaly) [reply Problem006/OCaml/solution_1.ml]: I didn't notice this before ! thanks 👍

**Comment by rafaelcn** [comment]
Weird, travis is giving an error on the line that I have requested a change. It should be working though.

**Comment by A-Hilaly** [comment]
the github new feature made something like:

```ocaml
let __kappa__ =
let __kappa__ = solve 100;;
```

It should be working now 🎉

**Comment by rafaelcn** [comment]
Wow, I didn't know that it duplicated code. Maybe that's because of the way I suggested it. Anyway, it's working now, you may replace the default image with the new one.

**Comment by A-Hilaly** [comment]
@rafaelcn Alright ! i'll update the default image and modify the `docker-compose.yml`

**Pull request merged**